gitpod-io/gitpod
# [jetbrains launcher] introduce backward compatible warmup command

## Description
Introducing a `warmup` cmd in `status` component (in the futrue it'll be renamed to `jetbrains-launcher` ) as a first step to land this PR: https://github.com/gitpod-io/gitpod/pull/15233/ with the end goal of moving the warmup logic in the jetbrains component instead of being in `server` or `ide-service`.

## Related Issue(s)

Relates to6740 #

## How to test
1. Start a workspace in prev env for IntelliJ stable
2. Make sure it works

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
